docs/source/contributing/docs.rst

```diff
@@ -0,0 +1,76 @@
+====================
+Adding Documentation
+====================
+
+Building the documentation
+--------------------------
+
+When you clone the Manim repository from GitHub, you can access the
+``docs/`` folder which contains the necessary files to build the
+documentation.
+
+To build the docs locally, enter the ``docs/`` folder in a CLI and
```

Comment:
This could perhaps be rephrased to something like "`cd` into the `docs/` folder" for clarity?
This way, if someone doesn't know how to do this, they could at least search up `cd`.

Reply:
Done! I mentioned `cd`.